networks/p2p/server.go

```diff
@@ -56,15 +56,20 @@ const (
 	frameWriteTimeout = 20 * time.Second
 )
 
-// TODO-Klaytn-Node Below constants are duplicated with node packages. Use it temporarily until fix cycle import problem.
+type ConnType int
+
 const (
-	CONSENSUSNODE = iota
+	ConnTypeUndefined ConnType = iota
+	CONSENSUSNODE
 	ENDPOINTNODE
 	PROXYNODE
 	BOOTNODE
 	UNKNOWNNODE // For error case
 )
 
+// TODO-Klaytn-Node Below constants are duplicated with node packages. Use it temporarily until fix cycle import problem.
+const ()
```

Comment:
What does it for?

Reply:
I missed removing the lines. I've removed this.